README.md

```
@@ -94,11 +94,21 @@ sudo pacman -S --needed base-devel ncurses
 brew install ncurses automake autoconf gcc
 ~~~
 
+**FreeBSD/Termux**
+~~~ shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)